bin/punic-build-data

```diff
@@ -1056,7 +1056,7 @@ class SourceData
             try {
                 $output = [];
                 $rc = null;
-                @exec('ant -f ' . argToShell($this->getRepositoryDirectory() . '/tools/java/build.xml') . ' jar 2>&1', $output, $rc);
+                @exec('mvn --settings mvn-settings.xml package -DskipTests=true --file ' . $this->getRepositoryDirectory() . '/tools/pom.xml 2>&1', $output, $rc);
```

Comment:
Where's the `mvn-settings.xml` file? It's [git-ignored](https://github.com/punic/data/pull/12/commits/2eae5a3aab88013d0579838a805abca948ff1a07#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R4)

Reply:
You need to make a local copy with your GitHub credentials based on mvn-settings.xml.dist.

I wonder if https://github.com/unicode-org/cldr-json has reached sufficient quality so that we can just check out the JSON files from there instead of building them locally.

Reply:
Instructions for Maven build including GitHub credentials: http://cldr.unicode.org/development/maven

Reply:
Do we have to create it as described at http://cldr.unicode.org/development/maven ?

Reply:
> I wonder if https://github.com/unicode-org/cldr-json has reached sufficient quality so that we can just check out the JSON files from there instead of building them locally.

Well, we used to use the built JSON files, but [we had to switch to building them on our own](https://github.com/punic/punic/pull/90) because of problems.
I don't think it's an issue to build them, we are sure things are done right :wink: